VeroliBGV/BGV003.xml

```diff
@@ -659,71 +654,73 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                      <decoDesc>
                         <decoNote type="band" xml:id="d1">
-                           <desc> Simple ornamental band in black and brown with a face is drawn on the top of <locus target="#4r"/> (incipit page of <ref target="#p2_i1"/>). Others of
-                              similar kind are found on <locus target="#76vb #148vb"/>. Short
-                              ornamental bands are occasionally drawn at the end of some subsections
-                              (e.g. <locus target="#7va #8ra #18ra #20ra"/> et passim). All those
-                              elements were most probably added at a later stage. </desc></decoNote>
+                           <desc> Simple ornamental band in black and brown with a face is drawn on
+                              the top of <locus target="#4r"/> (incipit page of <ref target="#p2_i1"
+                              />). Others of similar kind are found on <locus target="#76vb #148vb"
+                              />. Short ornamental bands are occasionally drawn at the end of some
+                              subsections (e.g. <locus target="#7va #8ra #18ra #20ra"/> et passim).
+                              All those elements were most probably added at a later stage.
+                           </desc></decoNote>
                      </decoDesc>
 
-
                      <additions>
                         <list>
                            <item xml:id="a1">
                               <locus from="168va" to="169ra"/>
-                              <desc type="GuestText"> <title ref="LIT6195PrayerSerpent" type="complete"/>Written by a professional scribe, whose writing
-                                 style is very similar to that of the main hand. The name of the ‘servant’
-                                 is erased.
-                              </desc>
+                              <desc type="GuestText">
+                                 <title ref="LIT6195PrayerSerpent" type="complete"/>Written by a
+                                 professional scribe, whose writing style is very similar to that of
```

Comment:
"whose writing style" you can simply write "whose handwriting" ref. https://www.lexico.com/definition/handwriting (def. 1.1)

Reply:
I think we would need an English proof reading in any case. However, I followed you suggestions and committed the change. Thank you for your comments!!!